src/sections/Community/Handbook/community.js

```diff
@@ -3,29 +3,239 @@ import { Container, Row, Col } from "../../../reusecore/Layout";
         <Container>
           <div className="content">
-            <h2>Where do I start? Who should I talk to?</h2>
-            <p>We realize that finding your way in a maze of fast-paced projects can be baffling, so we have an amazing program called “The Meshmate Program”. A <Link to="/community/meshmates">MeshMate </Link> is an experienced Layer5 community member, who will be available to guide you as you explore the various projects and find your areas of interest.</p>
-            <h3>Become a Meshtee today!	</h3>
-            <p>We love you to fill the Community Member Form and let us know you want a Meshamate. Then you can go ahead to review each MeshMate profile to identify your ideal mentor. Once you have identified your ideal MeshMate, reach out and introduce yourself in the #newcomers channel on Slack. Tell them about your areas of interest and focus in terms of languages and technology. The more they know about you, the more they can align project needs with your interests and abilities.</p>
-            <h2>Why contribute?</h2>
-            <p>Contributing to Open Source is an excellent way for gaining knowledge, engaging with the community, enhancing your skill set, and networking with different people across the world. As a contributor, working on a project like Meshery expands your experience with Docker, Kubernetes, Istio, Linkerd, Envoy, Octarine, App Mesh, Consul, Kuma, Maesh, and so on. With other amazing projects, you will also get to work with experienced software engineers in languages the projects use like Golang, Rust, Javascript (React, Gatsby, Nextjs, BillboardJs). In addition, contributors can get to work with DevOps tooling, continuous integration pipelining, documentation, and community management.</p>
+            <h2 id="communityGuidelines">Community Guidelines</h2>
+            <p>
+              Welcome to the Layer5 community! We’re happy to have you here and handhold you for your first contribution. Here are some tips to help you get started:
+            </p>
+            <p>
+              <img className="logo" src={Point} />
+              <strong>For Easy Identification -</strong> When setting up your slack workspace, your display name should be the same as your GitHub username for easy identification,
+              engage freely, and in whatever form you will: as a listener, user, contributor, maintainer, ambassador, or bystander. Involvement in any way is warmly received.
+            </p>
+            <p>
+              <img className="logo" src={Point} />
+              <strong>Familiarize with the community -</strong> As you get to familiarize yourself with the community, know that there are multiple initiatives for you to explore.
+              Some initiatives have working groups that meet at different times each week, while others use the weekly community meeting as their place of discussion. Details of each can be found on the <a href="https://bit.ly/2SbrRhe">community calendar</a>.
```

Comment:
instead of the <a> tag here could you use the <Link> tag and make it redirect to the community calendar page here: https://layer5.io/community/calendar